src/Provider.js

```diff
@@ -1,16 +1,16 @@
-import React, { Component, PropTypes as T, Children } from 'react';
-import { AlgoliaSearchHelper } from 'algoliasearch-helper';
+import {Component, PropTypes, Children} from 'react';
 
-import createStore from './createStore';
-import storeShape from './storeShape';
+import createStore from './createStore.js';
+import storeShape from './storeShape.js';
 
 class Provider extends Component {
   static propTypes = {
-    helper: T.instanceOf(AlgoliaSearchHelper),
+    helper: PropTypes.object.isRequired,
+    children: PropTypes.element.isRequired
```

Comment:
The correct way PropType for `children` is `PropTypes.node`, since we could be passed an array with only one value. `React.only(this.props.children)` already ensures that we only get passed one child.